### PR TITLE
Use idToken for renewal

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/Login.ts [hunk 1]
```diff
@@ -331,14 +344,35 @@ export class Login {
     });
   }
 
-  private static async verifyAccessToken(
-    accessToken: string
+  private static async verifyTokens(
```
Comment (valgussev) [inline src/Login.ts:347]: I would rename this in something more meaningful since we are never verify accessToken anymore

**Review by valgussev — APPROVED** [review]

File: src/Login.ts (hunk 1 above)
Comment (ghost) [reply src/Login.ts:347]: What do you mean? Which name?

Comment (valgussev) [reply src/Login.ts:347]: In `verifyTokens` method only `idToken` is verified. `accessToken` just passed in and out without the verfication, so it seems a method name is missleading

Comment (ghost) [reply src/Login.ts:347]: No, both are verified? https://github.com/cognitedata/cognitesdk-js/pull/58/files/75ff0a680cd54bff4db586965e7b7386468b9982#diff-44266178f674855837c2cb06417aec76R361

Comment (ghost) [reply src/Login.ts:347]: Actually the combination is verified

**Pull request merged**